[2024-10-06T18:04:57.707+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-06T18:04:58.682+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T01:04:01+07:00 [queued]>
[2024-10-06T18:04:58.710+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T01:04:01+07:00 [queued]>
[2024-10-06T18:04:58.720+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-06T18:04:58.821+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_to_sqlite> on 2024-10-06 18:04:01+00:00
[2024-10-06T18:04:58.848+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=137) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-06T18:04:58.857+0000] {standard_task_runner.py:72} INFO - Started process 143 to run task
[2024-10-06T18:04:58.902+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_to_sqlite', 'manual__2024-10-07T01:04:01+07:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp__4_vvrx']
[2024-10-06T18:04:58.920+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask load_to_sqlite
[2024-10-06T18:04:59.202+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T01:04:01+07:00 [running]> on host 17f17543b6e1
[2024-10-06T18:04:59.492+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2024-10-06T18:04:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T01:04:01+07:00'
[2024-10-06T18:04:59.502+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-06T18:05:03.810+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-06T18:05:03.950+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-06T18:05:03.952+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_to_sqlite, run_id=manual__2024-10-07T01:04:01+07:00, execution_date=20241006T180401, start_date=20241006T180458, end_date=20241006T180503
[2024-10-06T18:05:04.129+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-06T18:05:04.182+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-06T18:05:04.187+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
